lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -1276,10 +1328,20 @@ void SessionState::get_charging_updates(
         }
         grant->set_service_state(SERVICE_REDIRECTED, *credit_uc);
         action->set_redirect_server(grant->final_action_info.redirect_server);
+      case RESTRICT_ACCESS: {
+        if (grant->service_state == SERVICE_RESTRICTED) {
+          MLOG(MDEBUG) << "Service Restriction is already activated.";
```

Comment:
can we make the log similar to `REDIRECT` 
`Restriction already activated.`
(or reverse, change the REDIRECT log)

homogeneity helps when skimming logs 

Also it may be useful to add the sessionID to both messages